src/components/header.js

```diff
@@ -88,8 +88,8 @@ const Header = props => (
         font-family: 'Open Sans', sans-serif !important;
       `}
     >
-      This website is kept up-to-date with climate data provided by NASA and
-      Scripps Institution of Oceanography
+      This website is kept up-to-date with climate data provided by <a href="https://www.nasa.gov/" target="_blank" rel="noopener">NASA</a> and
+      <a href="https://scripps.ucsd.edu/" target="_blank" rel="noopener">Scripps Institution of Oceanography</a>
```

Comment:
There is no space rendering before the "Scripps Institution of Oceanography" link. You might need to add `{" "}` before the `<a>` tag to add a space there.